qutip/fileio.py

```diff
@@ -243,23 +245,14 @@ def qload(name):
         Object retrieved from requested file.
 
     """
-    with open(name + ".qu", "rb") as fileObject:
+    file = Path(name)
+    if file.suffix != '.qu':
+        file.with_suffix('.qu')
+
+    with open(name, "rb") as fileObject:
```

Comment:
The behaviour here has changed as it no longer automatically includes the .qu. I believe this was meant to be dealt with by the file.with_suffix above, but the file Path object is never used.